### PR TITLE
TRSv2-491 - Request organisation details for rep from new org

### PR DESCRIPTION
**Pull request by nahmed99**

### CODE REVIEW

**Review by osimuka — APPROVED** [review]

File: trade_remedies_public/cases/v2_views/accept_representative_invitation.py [hunk 1]
```diff
@@ -61,45 +61,56 @@ def form_valid(self, form):
...
-            confirmed=True,
-            fields=["id"],
-        )
-
-        # now let's validate the person's email
-        return redirect(
-            reverse(
-                "request_email_verify_code", kwargs={"user_pk": self.invitation.invited_user.id}
+            # now let's validate the person's email
+            return redirect(
```
Comment (chris-pettinga) [inline trade_remedies_public/cases/v2_views/accept_representative_invitation.py:108]: We need to move the if statement down here. We still want all the above stuff to happen if the organisation has just been created. The only difference is where we go next.

**Review by chris-pettinga — CHANGES_REQUESTED** [review]

File: trade_remedies_public/cases/v2_views/accept_representative_invitation.py (hunk 1 above)
Comment (nahmed99) [reply trade_remedies_public/cases/v2_views/accept_representative_invitation.py:108]: Most of the stuff above this point will happen in the two classes below (that's where this code was copied from) - while we are gathering the organisation (+additional) details - for the new organisation.

I know that there is duplication of code. Perhaps another method to merge some of that code?

Comment (chris-pettinga) [reply trade_remedies_public/cases/v2_views/accept_representative_invitation.py:108]: hmm

Comment (chris-pettinga) [reply trade_remedies_public/cases/v2_views/accept_representative_invitation.py:108]: ok looking at the view, it seems like I have scattered the invitation processing stuff around pretty poorly. So let's shelve that for another ticket and go ahead with the duplication for the moment so we can push this through to UAT

**Review by chris-pettinga — APPROVED** [review]

**Pull request merged**